utils/superset/import_template.json

```diff
@@ -0,0 +1,5910 @@
+{
```

Comment:
I think renaming this to make it clearer what the dashboard does would be good. pgd_monitoring_template.json?

Reply:
Done.